compiler/damlc/daml-ide-core/dlint.yaml

```diff
@@ -663,6 +663,10 @@
 
 # Not popular or useful
 - ignore: {name: Use camelCase}
+# Interacts badly with desugaring and the hack for
+# -Wunused-matches. Also, it's generally considered by Neil to be
+# dodgy.
+- ignore: {name: Reduce duplication}
```

Comment:
Totally agree that disabling this is sensible. IME a lot of DAML projects have already disabled this since it’s pretty aggressive and triggers even for fairly tiny amounts of duplication.